ipc/provider/src/manager/evm/manager.rs

```diff
@@ -485,16 +512,8 @@ impl SubnetManager for EthSubnetManager {
         Ok(TokenAmount::from_atto(balance.as_u128()))
     }
 
-    async fn last_topdown_executed(&self, gateway_addr: &Address) -> Result<ChainEpoch> {
-        self.ensure_same_gateway(gateway_addr)?;
-
-        let gateway_contract = gateway_getter_facet::GatewayGetterFacet::new(
-            self.ipc_contract_info.gateway_addr,
-            Arc::new(self.ipc_contract_info.provider.clone()),
-        );
-        let epoch = gateway_contract.last_voting_executed_epoch().call().await?;
-
-        Ok(epoch as ChainEpoch)
+    async fn last_topdown_executed(&self, _gateway_addr: &Address) -> Result<ChainEpoch> {
```

Comment:
Does this still exists with top-down proofs of finality? If this is the case we may want to rename the function.